Kconfig

```diff
@@ -10,6 +10,13 @@ config CONFIGURATION_FILE_NAME
 	help
 		Configuration file name
 
+config DOCKER_FILE_NAME
```

Comment:
I think we can shorten the length of `CONFIGURATION_FILE_NAME` and `DOCKER_FILE_NAME` such as `CONFIGFILE` and `DOCKERFILE`. What do you think?

Reply:
Ok, I'll do it!

Reply:
Done!